### PR TITLE
Link to BUILDING.md for testing instructions

### PR DESCRIPTION
**Pull request by ikedam**
**What I did**

Link to BUILDING.md for testing instructions, which was linked to README.md.
(BUILDING.md wasn't linked from anywhere)

**Related issue**

#9439

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Fork the repository and make changes on your fork in a feature branch:
 
 Submit unit tests for your changes. Go has a great test framework built in; use
 it! Take a look at existing tests for inspiration. [Run the full test
-suite](README.md) on your branch before
+suite](BUILDING.md) on your branch before
```
Comment (glours) [inline CONTRIBUTING.md:128]: Why not link directly to the right section?
```suggestion
suite](BUILDING.md#end-to-end-tests) on your branch before
```

Comment (ikedam) [reply CONTRIBUTING.md:128]: Contributors are expected to run both unit tests and e2e tests. Linking to the whole document makes sense for that reason.

Which is the best?:

* A: Link to `BUILDING.md#unit-tests` as this section describes "Submit unit tests".
* B: Rewrite this section. E.g. `Run the full test suite, both unit tests and e2e tests on your branch before submitting a pull request. See BUILDING.md for instructions to build and run tests.`

Comment (nicksieger) [reply CONTRIBUTING.md:128]: I'm fine with the change as-is, it's a small improvement. I do like the rewrite option B that @ikedam puts here too.

Comment (ikedam) [reply CONTRIBUTING.md]: Thanks for the comment.
I added a new commit updating this paragraph as option B.

**Review by glours — APPROVED** [review]

**Pull request merged**